tests/test_utils.py

```diff
@@ -192,16 +192,14 @@ def required_extensions(self):
 
 if os.environ.get('TI_LITE_TEST', ''):
     _test_features = {
-        "dynamic_index": [TestParam(False, [])],
+        "dynamic_index": [TestParam(True, [])],
```

Comment:
Do we still need these code since it's automatically on?

Reply:
Nope. I had an incorrect assumption that `_test_features` had to be non-empty :-)

Fixed with only one commented line left to serve as an example for potential future use.